### PR TITLE
Link to the reworkcss AST explorer

### PR DESCRIPTION
**Pull request by iamdustan**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -77,6 +77,8 @@ properties for consistency.
 
 ## AST
 
+Interatively explore the AST with [http://iamdustan.com/reworkcss_ast_explorer/](http://iamdustan.com/reworkcss_ast_explorer/)
```
Comment (conradz) [inline Readme.md]: Nit: missing period

**Comment by conradz** [comment]
Wow, that really looks cool! I'm :+1: on merging this.

File: Readme.md (hunk 1 above)
Comment (iamdustan) [inline Readme.md]: Added. I always go back and forth on periods after URLs for one-off statements. Granted, in proper english that’s not a debate :)

**Comment by iamdustan** [comment]
Thanks! Fwiw, it’s a pretty straight fork of https://github.com/fkling/esprima_ast_explorer

File: Readme.md [hunk 2]
```diff
@@ -77,6 +77,8 @@ properties for consistency.
 
 ## AST
 
+Interatively explore the AST with [http://iamdustan.com/reworkcss_ast_explorer/](http://iamdustan.com/reworkcss_ast_explorer/).
+
```
Comment (lydell) [inline Readme.md]: You could use markdown's autolink syntax here: `<http://iamdustan.com/reworkcss_ast_explorer/>`.

Comment (iamdustan) [inline Readme.md]: Thanks for the tip, @lydell! Do you know off-hand if that standard markdown or github extensions?

Do you want me update that before landing this?

Comment (lydell) [inline Readme.md]: That's standard markdown. Yes, please amens your commit.

**Comment by iamdustan** [comment]
Updated and rebased.

**Comment by ai** [comment]
AST explorer is awesome!

**Comment by iamdustan** [comment]
Thanks @ai. I will be pushing some updates over the weekend to fix a number of bugs and features it’s missing from the original project.

**Comment by conradz** [comment]
Thanks again for this, I'm sure it will help everyone using this library!

**Pull request merged**